python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -87,6 +87,7 @@ class GrapheneAssetStaleStatusCause(graphene.ObjectType):
     status = graphene.NonNull(GrapheneAssetStaleStatus)
     key = graphene.NonNull(GrapheneAssetKey)
     reason = graphene.NonNull(graphene.String)
+    dependency = GrapheneAssetKey
```

Comment:
This might have to be `graphene.Field(GrapheneAssetKey)`? Didn't know this could be a bare property actually 🙌